docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -22,6 +22,14 @@ On the consumer side, there are some common variations:
 
 The source of the events may be external to the system, such as physical devices in an IoT solution. In that case, the system must be able to ingest the data at the volume and throughput that is required by the data source.
 
+There are two primary approaches to structuring event payloads:
+
+- **Including all required attributes in the payload**: This approach is faster and more scalable, as consumers do not need to query the database. However, it can lead to data consistency issues due to multiple [systems of record](https://en.wikipedia.org/wiki/System_of_record), particularly after updates. Additionally, this method may result in [stamp coupling](https://en.wikipedia.org/wiki/Coupling_(computer_programming)) and bandwidth challenges at scale. Contract management and versioning can also become complex.
+
+- **Including only key(s) in the payload**: In this approach, consumers retrieve the necessary attributes by querying the database. While this method offers better data consistency due to a single system of record, it is less scalable and performs poorer than the first approach since consumers must query the database frequently. There are fewer concerns regarding stamp coupling, bandwidth, contract management, or versioning, as events are smaller and contracts simpler.
```

Comment:
```suggestion
- **Including only key(s) in the payload**: In this approach, consumers retrieve the necessary attributes, such as a primary key, to independently fetch the remaining data from a data source. While this method offers better data consistency due to a single system of record, it can perform poorer than the first approach since consumers must query the data source frequently. There are fewer concerns regarding coupling, bandwidth, contract management, or versioning, as events are smaller and contracts simpler.
```